.github/workflows/ci.yml

```diff
@@ -62,15 +62,9 @@ jobs:
         run: Scripts/build.swift spm ${{ matrix.platform }}
   carthage:
     name: Carthage Build
-    runs-on: macOS-11
-    strategy:
-      matrix:
-        platform: ['iOS_14']
-      fail-fast: false
+    runs-on: macOS-13
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v3
-      - name: Select Xcode Version (12.5.1)
-        run: sudo xcode-select --switch /Applications/Xcode_12.5.1.app/Contents/Developer
       - name: Build
-        run: Scripts/build.swift carthage ${{ matrix.platform }}
+        run: carthage build --no-skip-current --platform iOS --use-xcframeworks
```

Comment:
The macOS-13 build [already includes Carthage 0.39.0 as a dependency](https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md#package-management), so no need to install it first.